contracts/axelar-gateway/src/event.rs

```diff
@@ -25,8 +30,8 @@ pub(crate) fn execute_contract_call(env: &Env, message: Message) {
 }
 
 pub(crate) fn rotate_signers(env: &Env, signers_hash: BytesN<32>, epoch: u64) {
-    let topics = (symbol_short!("rotated"),);
-    env.events().publish(topics, (signers_hash, epoch));
+    let topics = (symbol_short!("rotated"), signers_hash, epoch);
```

Comment:
```suggestion
    let topics = (symbol_short!("rotated"), epoch, signers_hash);
```
check EVM gateway for what comes first

Reply:
Good call, newEpoch comes first on EVM gateway: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/2472998993d4c6c914d6cf676e8a7183c4a15f72/contracts/governance/BaseWeightedMultisig.sol#L160

updated.